### DISCUSSION TITLE
Leveraging Detectron2 for a Web Page UI Detection Project

### DISCUSSION THREAD

**Discussion by DeepTitan** [facebookresearch/detectron2#5034] [Q&A]
Hello everyone,

I am working on a project that involves the detection of user interface (UI) elements on web pages. I have been considering Detectron2 as the backbone for my object detection model and am eager to learn if anyone else has tried something similar.

Are there any developers who have previously employed Detectron2 for web page UI detection and achieved successful results? I'm interested in learning more about your methodologies and any challenges you may have encountered in the process.

Also, if there are any specific pre-processing techniques, training strategies, or post-processing methods that you found particularly beneficial for this specific application, I would greatly appreciate any advice or resources you could share.

Thank you in advance for your insights!

**Comment by RobYed** [comment]
Hey @DeepTitan I just found https://github.com/kvyb/Segmentation-of-web-UI-elements-with-Detectron2
Maybe it helps :)

**Comment by smyja** [comment]
> Hey @DeepTitan I just found https://github.com/kvyb/Segmentation-of-web-UI-elements-with-Detectron2
> Maybe it helps :)

Not maintained again, but it's great start.
